packages/components-react/projects/uxpin-wrapper/package.json

```diff
@@ -23,7 +23,7 @@
   "devDependencies": {
     "@babel/preset-typescript": "^7.18.6",
     "@types/react": "^18.0.25",
-    "@uxpin/merge-cli": "2.8.0-dev.77",
+    "@uxpin/merge-cli": "2.9.0",
```

Comment:
Please also update `docs/dependencies.md` by removing the entry related to this.

Reply:
Done here https://github.com/porsche-design-system/porsche-design-system/pull/2208/commits/959c331a17d0b714cfaf3aec8ccd5b04442b42b0

I'm always happy to clean stuff 😄